test_common/page_objects/et1/additional_respondents_details_page.rb

```diff
@@ -52,6 +52,7 @@ def remove_another_respondent
 
         def save_and_continue
           page.scroll_to(main_content.save_and_continue_button, align: :bottom)
+          sleep(0.5.second) #Save and continue button click randomly fails without half second wait after scrolling
```

Comment:
I always avoid sleeps like the plague.
The problem with the buttons are often because they are half on and half off the page - they sometimes fail - so scrolling fixes that.
Not sure if you can wait for something happening as opposed to waiting for half a second ?